### ISSUE TITLE
Tracking page views - pop-ups

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/thinkhazard#683]
In the google tracking capabilities, is it possible to identify how many times users open pop-ups for recommendations / navigate to geonode to download data?

E.g., when Google cites '/' or '/report' in user page flow, does that actually represent the viewing of a pop-up?

**Comment by pgiraud** [comment]
@stufraser1
By default, google analytics tracks pages. This means that each page loaded by the browser sends a new hit to google analytics with information like the url of the page.
However, it's possible to also track events like user interaction within a page.
https://developers.google.com/analytics/devguides/collection/analyticsjs/events

Currently we only track pages. The recommendation detailed text being displayed in a popup is not tracked.

**Comment by stufraser1** [comment]
In that case, please can you build in tracking to see when a user:
1. Opens up the FAQ popup window
2. Opens up the About popup window 
3. Opens up a detailed recommendation popup window (noting the country+hazard)
4. Clicks on contact list
5. Clicks on further resource

As an additional task, this can be included under support days if required

**Comment by stufraser1** [comment]
The WB Operations Portal (a WB internal site) directs project planners to thinkhazard.org, for further information. GFDRR to check whether Google Analytics tracks where users come from.

**Comment by fjacon** [comment]
@stufraser1 do you want us to go ahead with this ?

**Comment by stufraser1** [comment]
Yes please, it would be very helpful to understand how people are using these features, and at the moment I don't think we get that from Google analytics

**Comment by tonio** [comment]
@stufraser1 could you please grant me access to the analytics account, so that I can start testing?
